contracts/seed/Seed.sol

```diff
@@ -480,23 +488,23 @@ contract Seed {
         uint256 amountVestedPerSecond = _seedAmount.div(vestingDuration);
         require(amountVestedPerSecond > 0, "Seed: amountVestedPerSecond > 0");
 
-        tokenLocks[_recipient] = Lock({
+        funders[_recipient] = FunderPortfolio({
             seedAmount: _seedAmount,
             secondsClaimed: _secondsClaimed,
             totalClaimed: _totalClaimed,
             fundingAmount: _fundingAmount,
             fee: _fee,
             feeClaimed: _feeClaimed
             });
-        totalLockCount++;
+        totalFunderCount++;
     }
 
     /**
-      * @dev                     calculates claim for a lock
-      * @param _locker           Address of lock to calculate days and amount claimable
+      * @dev                     calculates claim for a funder
+      * @param _funder           Address of funder to calculate days and amount claimable
     */
-    function _calculateClaim(address _locker) private view returns (uint32, uint256) {
-        Lock memory tokenLock = tokenLocks[_locker];
+    function _calculateClaim(address _funder) private view returns (uint32, uint256) {
```

Comment:
I don't think elapsedSeconds is any more needed in the return value, neither in this function nor the public wrapper for it (actually why do we even need this private function at all?  just call the public function directly?)

Reply:
Because this will be used to get the maximum claim a user can do. It is used in a getter function, calculateClaim, which will give the maximum amount a user can claim at the moment.

Reply:
Also, fixed the seconds, now we don't return elapsedSeconds.

Reply:
There is no need to have _calculateClaim, am I wrong?  It is identical to calculateClaim.  I think we should move the contents of _calculateClaim into calculateClaim and delete _calculateClaim.